### DISCUSSION TITLE
Change lualine function location separators background?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#3459] [Q&A]
<img width="707" alt="image" src="https://github.com/LazyVim/LazyVim/assets/8405459/8753f73b-d5fc-442c-a368-8ecd3ac66ba9">

I am getting this black boxes next to the location from statusline, not sure when this started happening

**Comment by TheDenast** [comment]
I'm running a very default LazyVim config and suffer from the same issue. Not sure if it's a bug or a configuration mistake. 

I've tried changing lualine themes manually to see if the problem occurs when it tries to determine theme automatically, but it doesn't seem to be the case:

```lua
-- ~/.config/nvim/lua/plugins/lualine.lua

return {
  "nvim-lualine/lualine.nvim",
  opts = function(_, opts)
    opts.options.theme = "dracula"
  end,
}
```
Result:
![image](https://github.com/LazyVim/LazyVim/assets/99830990/ccf0de07-675a-4def-b5e5-48fb69d9162f)

```lua
-- ~/.config/nvim/lua/plugins/lualine.lua

return {
  "nvim-lualine/lualine.nvim",
  opts = function(_, opts)
    opts.options.theme = "gruvbox"
  end,
}
```
Result:
![image](https://github.com/LazyVim/LazyVim/assets/99830990/6662571e-0bab-4e9d-9053-c83b7188de51)

**Reply by gnakicta** [reply]
Noticed the same thing with kanagawa theme in my setup. Mentioned it in https://github.com/LazyVim/LazyVim/discussions/3583

**Reply by Raagh** [reply]
I endup removing the breadcrumbs from lualine to avoid seeing this, not ideal but okay.

**Reply by dpetka2001** [reply]
Make sure that in your colorscheme `lualine_c_normal` and `lualine_c_inactive` use the same background color.

**Reply by chronon** [reply]
I'm seeing this too. It appears that the black boxes are not part of the lualine theme. If I set:

```
highlight lualine_c_normal guibg=red
highlight lualine_c_inactive guibg=blue
```
the black boxes are still there.

![lualine](https://github.com/user-attachments/assets/c69f9dfc-d33b-42c8-b26d-a1adec31b65a)

**Reply by zombiearnie88** [reply]
> lualine

how can i remove breadcrumbs from lualine ? pls share your config.

**Comment by portnov** [comment]
It appears for these separators standard "StatusLine" group is used.
``
highlight StatusLine guibg=LightGray
``
